src/allotropy/parsers/moldev_softmax_pro/softmax_pro_structure.py

```diff
@@ -291,7 +292,7 @@ class DataElement:
     temperature: Optional[float]
     wavelength: float
     position: str
-    value: float
+    value: float | InvalidJsonFloat
```

Comment:
`JsonFloat` is an alias for `float | InvalidJsonFloat`

Reply:
Will get this in util PR as well